package.json

```diff
@@ -16,13 +16,13 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "bn.js": "^2.0.0",
-    "browserify-rsa": "^2.0.0",
+    "bn.js": "^3.0.0",
```

Comment:
why not `3.2.0`?  You have been switching between `3.2.0` and `3.0.0`.
Do we need `3.2.0`?